src/main/kotlin/lotto/Lotto.kt

```diff
@@ -0,0 +1,21 @@
+package lotto
+
+class Lotto(private val _lotto: Set<LottoNumber>) {
+
+    init {
+        validate(_lotto)
+    }
+
+    val numbers: Set<LottoNumber>
+        get() = _lotto.toSet()
```

Comment:
```suggestion
class Lotto(val numbers: Set<LottoNumber>) {

    init {
        validate(numbers)
    }
```
내부적으로 데이터를 변경하는 것이 아니기 때문에, 굳이 private를 쓸 필요가 없어보여요. (이하 관련 내용 동일)